starport/pkg/chaincmd/runner/runner.go

```diff
@@ -114,7 +121,7 @@ func (r Runner) run(ctx context.Context, roptions runOptions, soptions ...step.O
 		stderr = io.MultiWriter(stderr, roptions.stderr)
 	}
 
-	if !roptions.longRunning {
+	if roptions.wrapStdErr {
```

Comment:
Previously, this was `true` for all short running commands like `gentx` which allow us to capture any errors from those commands. Right now, this line only allows long running commands instead. We should also allow it for short running commands.

Reply:
Yes I changed the name because the difference with longRunning was only about capturing StdErr or not but I didn't pay attention the condition got reversed

What about having the option named `IgnoreStdErr` and reversing the condition? Renaming it to long running would be strange since `start` is long runningbut we need to capture stderr

Reply:
Hm, I think we can completely remove `wrapStdErr` and that should do it. We need to collect errors in all cases regardless of command is long running or not.

Reply:
Ok that also works for me